### DISCUSSION TITLE
Want to change the drop chance of backpacks on mobs

### DISCUSSION THREAD

**Discussion by MemeKueen2** [P3pp3rF1y/SophisticatedBackpacks#1061] [Q&A]
specifically when they're killed by things other than the player. even more specifically, I have a gold farm (ilmango's design) and I want the backpacks the zombie piglins spawn with to drop but they don't drop unless the player kills them, and the farm design uses entity cramming to kill the zombie piglins.

that being said I cannot for the life of me figure out how to change the config file. So I either need a detailed explanation on how to do so, or if someone would be so kind as to do it and then send me the file, that would be much appreciated. I have the mod installed through curseforge btw, not sure if that's why I'm having a difficult time trying to figure it out (cuz of the way curseforge stores files)

**Comment by P3pp3rF1y** [comment]
there's a setting to allow drops for fake players in config and so you would want to change the killing to use some kind of fake player in your case (like mob grinding utils grinder). I am not really in favor of making backpacks drop even in case that non player (not even fake) is the cause of the death. That would mean that something like sun killing undead would have a chance of dropping backpack and that just sounds way too easy for something that requires a bit of materials and many times includes valuable loot.
